blog/2025/how-to-fetch-asset-price-using-dia-oracle.md

```diff
@@ -0,0 +1,72 @@
+---
+category: 2025
+date: 2025-02-12
+seo:
+    title: How to Fetch Up-to-Date Asset Price Information Using DIA Oracles
+    description: Explore ecosystem price oracles such as DIA and discover how to use DIA Oracles on the XRP Ledger to fetch up-to-date asset price information. 
+labels:
+    - Development
+markdown:
+    editPage:
+        hide: true
+---
+# How to Fetch Asset Price Information Using DIA Oracles
+
+Explore ecosystem price oracles such as DIA and discover how to use DIA Oracles on the XRP Ledger to fetch up-to-date asset price information. 
+
+<!-- BREAK -->
+
+## Introduction to DIA
+
+[DIA](https://www.diadata.org/) is a trustless oracle network that delivers data feeds for any token, LST, RWA, random numbers and more to any blockchain. The network is permissionless, enabling nodes and stakers to participate in distributed data sourcing, secured by crypto-economic and cryptographic mechanisms. DIA’s native Ethereum L2 rollup, ‘Lasernet’, executes all critical oracle operations, ensuring trustless computation and full verifiability.
+
+## Usage of DIA Oracles on XRP Ledger
+
+dApps building on XRP Ledger can utilize DIA oracles to obtain up-to-date asset price information. These deployed oracles are suitable for use in production environments. They come with a list of supported assets and settings. However, if dApps require a custom oracle with a different set of assets and configurations, they should [contact DIA on Telegram](https://t.me/diabdteam).
+
+## Deployed Contracts
```

Comment:
Is this the correct term? As far as I know oracles are just a ledger entry.

Reply:
It could be renamed as "Oracle Document IDs"